zmsbackend/src/Zmsbackend/Request/Repository/Request.php

```
@@ -59,6 +59,39 @@ public function addConditionProcessId($processId)
+            'buergeranliegen.BuergerID',
+            array_values(array_map('intval', $processIds))
+        );
+        $this->query->orderBy('buergeranliegen.BuergerID', 'ASC');
+        $this->query->orderBy('buergeranliegen.BuergeranliegenID', 'ASC');
+        return $this;
+    }
+
+    public function addEntityMappingWithProcessId($type = null): self
+    {
+        $entityMapping = $this->getPrefixedList(array_merge(
+            $this->getEntityMapping($type),
```

PHPStan (2.2.6)

[error] Method BO\Zmsbackend\Request\Repository\Request::getEntityMapping() invoked with 1 parameter, 0 required.


(arguments.count)